src/placement/pl_map_common.c

```diff
@@ -287,23 +287,23 @@ determine_valid_spares(struct pool_target *spare_tgt, struct daos_obj_md *md,
 		 */
 		if (spare_tgt->ta_comp.co_fseq > md->omd_ver) {
 			D_DEBUG(DB_PL, DF_OID", "DF_TARGET", ver: %d\n",
-				DP_OID(md->omd_id),
-				DP_TARGET(spare_tgt),
+				DP_OID(md->omd_id), DP_TARGET(spare_tgt),
 				md->omd_ver);
 			spare_avail = false;
 			goto next_fail;
 		}
 
+		if (spare_tgt->ta_comp.co_in_ver < md->omd_ver &&
+		    spare_tgt->ta_comp.co_status == PO_COMP_ST_UP)
```

Comment:
you add this "if" here and without change indent of below section? some comment maybe helpful, thanks.

Reply:
ah, this should be removed. Thanks!